app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -14,6 +13,15 @@ def new
 
     def create
       @contact_method = params[:contact_method] unless @contact_method.present?
+      @form = request_login_form_class.new(request_client_login_params)
+      if @form.valid?
+        intake_classes = client_login_service.intake_classes
+        @records = intake_classes.map { |intake_class| @form.filter_records(intake_class) }.flatten
+        if @records.blank?
+          flash[:alert] = I18n.t("state_file.intake_logins.new.#{@contact_method}.not_found")
+          render :new and return
+        end
+      end
```

Comment:
If the form is valid, we check that there is an existing intake. I figured it made sense to do DB lookups in a controller rather than a validator - please LMK if you disagree.